spaceR/spaceR/Controllers/RocketsControlller.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  RocketsControlller.swift
+//  spaceR
+//
+//  Created by Kieran Woodrow on 2022/02/25.
+//
+
+import Foundation
+
+
+import UIKit
+
+class RocketsController: UIViewController {
+    @IBOutlet weak var tableView: UITableView!
+    //Creates an array of rockets.
```

Comment:
You can remove these comments, as I mentioned above we don't really like to commit comments 😄

Reply:
will do